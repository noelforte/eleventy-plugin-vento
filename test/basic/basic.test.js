import { runBuild } from '../get-instance.js';
import { expect, test } from 'vitest';

const results = await runBuild(import.meta.dirname);

test('renders a basic template', async () => {
	await expect(results[0].content).toMatchSnapshot();
});
