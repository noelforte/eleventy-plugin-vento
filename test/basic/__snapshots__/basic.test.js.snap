// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders a basic template 1`] = `
"
<h1>Basic Page</h1>

This is a standalone page.
"
`;
